web/.eslintrc.json

```diff
@@ -4,6 +4,7 @@
   "rules": {
     "arrow-parens": ["error", "as-needed"],
     "comma-dangle": [1, "never"],
+    "no-confusing-arrow": 0,
```

Comment:
Why would we disable this rule?  We're adding a lot of customization to our default lint ruleset.  I'm not a big fan of that.

Reply:
keep in mind that most of these are because prettier is better than eslint :)

Reply:
That, sir, is an opinion.  😉

Reply:
re: `no-confusing-arrow`, this comes up in the render prop here:

```
  <Route
    {...rest}
    render={props =>
      authenticated ? (
        <Component {...props} />
      ) : (
        <Redirect
          to={{
            pathname: '/login',
            state: { from: props.location }
          }}
        />
      )
    }
  />
```

by disabling, we avoid having to include repetitive `return` statements and because prettier formats it nicely, i don't believe this is "confusing".